docs/books/user-guide/routing.adoc

```diff
@@ -551,6 +562,11 @@ Link routes establish a link between a sender and a receiver that travels throug
 
 With link routing, client traffic is handled on the broker, not the router. Clients have a direct link through the router to a broker's queue. Therefore, each client is a separate producer or consumer.
 
+[NOTE]
+====
+Even though the client has a direct link to the broker, it will not be able to see a connection failure if the broker goes offline and will not attempt to reestablish the connection when the broker comes back online. If you want the connection to be reestablished automatically in this type of scenario, then you should use message routing with autolinks instead of link routing. For more information, see xref:routing-messages-through-broker[Routing Messages through a Broker Queue] and xref:considerations-for-broker-connection-failure-handling[Considerations for Broker Connection Failure Handling].
```

Comment:
This is not correct.  Connection reestablishment is _always_ attempted.  With link-routing, the client needs to handle dropped links rather than (in addition to) dropped connections.  With message-routing, the client needs to handle RELEASED/MODIFIED deliveries (usually by re-sending them).